packages/open-truss/src/configuration/apply.tsx

```diff
@@ -38,3 +40,19 @@ export function applyConfiguration(
 
   return configurationFunction
 }
+
+export async function RenderFromFile({
+  components = {},
+  path,
+}: {
+  components?: COMPONENTS
+  path: string
+}): Promise<JSX.Element> {
+  const config = await fs.readFile(path, 'utf-8')
+  // TODO: Render 404 if no file
```

Comment:
See https://github.com/open-truss/open-truss/pull/57#discussion_r1432205144, I'm not sure how to handle this yet. Maybe this can't be in the OT package? That would be sad though.

Reply:
Yes so in this case `fs.readFile` will throw and the application could catch that error and do what it wants 👍 That seems fine to me!